cli_tests/test_list_debuggess_command.py

```diff
@@ -0,0 +1,205 @@
+            'version': 'v1'
+        },
+        'description': 'desc 1'
+    }
+    self.debuggee1_table_values = ['app123 - v1', '123', 'desc 1']
+    self.debuggee1_json_value = self.debuggee1
+
+    self.debuggee2 = {
+        'id': '456',
+        'labels': {
+            'module': 'app456',
+            'version': 'v2'
+        },
+        'description': 'desc 2'
+    }
+    self.debuggee2_table_values = ['app456 - v2', '456', 'desc 2']
+    self.debuggee2_json_value = self.debuggee2
+
+    self.debuggee3 = {
+        'agentVersion': 'google.com/node-gcp/v6.0.0',
+        'description': 'node index.js module:test-app version:v1',
+        'id': '97b0090602ceb5a506f1cf7dff02524f',
+        'labels': {
+            'V8_version': '7.8.279.23-node.56',
+            'agent_name': '@google-cloud/debug-agent',
+            'agent_version': '6.0.0',
+            'main_script': 'index.js',
+            'module': 'test-app',
+            'node_version': '12.22.5',
+            'platform': 'default',
+            'process_title': 'node',
+            'projectid': 'unknown',
+            'version': 'v1'
+        },
+        'project': 'unknown',
+        'uniquifier': 'ecbb07aaba5ad405ad7667bf3eacbd874e23a2f5'
+    }
+
+    self.debuggee3_table_values = [
+        'test-app - v1', '97b0090602ceb5a506f1cf7dff02524f',
+        'node index.js module:test-app version:v1'
+    ]
+    self.debuggee3_json_value = self.debuggee3
+
+  def test_get_debuggees_called_once(self):
+    args = self.args_parser.parse_args(['list_debuggees'])
+    self.rtdb_service_mock.get_debuggees = MagicMock(return_value={})
+
+    self.list_debuggees.cmd(args, self.cli_services)
+    self.rtdb_service_mock.get_debuggees.assert_called_once()
+
+  def test_output_format_default(self):
+    expected_headers = ['Name', 'ID', 'Description']
+
+    testcases = [({}, []), ({
+        '1': self.missing_id
+    }, self.missing_id_table_values),
+                 ({
+                     self.missing_labels['id']: self.missing_labels
+                 }, [self.missing_labels_table_values]),
+                 ({
+                     self.missing_description['id']: self.missing_description
+                 }, [self.missing_description_table_values]),
+                 ({
+                     self.debuggee1['id']: self.debuggee1
+                 }, [self.debuggee1_table_values]),
+                 ({
+                     self.debuggee1['id']: self.debuggee1,
+                     self.debuggee2['id']: self.debuggee2,
+                 }, [self.debuggee1_table_values, self.debuggee2_table_values]),
+                 ({
+                     self.debuggee1['id']: self.debuggee1,
+                     self.debuggee2['id']: self.debuggee2,
+                     self.debuggee3['id']: self.debuggee3
+                 }, [
+                     self.debuggee1_table_values, self.debuggee2_table_values,
+                     self.debuggee3_table_values
+                 ])]
```

Comment:
This auto-formatting is regrettable.  Is there a way to clean this up?

Reply:
Thanks for prompting me to look, turns out the answer is yes, you can disable/enabled yapf over blocks of code.